src/docs/product/discover-queries/query-builder.mdx

```diff
@@ -85,20 +85,22 @@ When the "Display" selection is either "Top Period" or "Top Daily", an optional
 
 When your "Display" selection is either "Total Period", "Previous Period", "Total Daily" or "Bar Chart", you can set up to three y-axes by selecting multiple checkboxes in the "Y-Axis" dropdown.
 
-## Filter by Table Columns
+## Group & Stack With Table Columns {#filter-by-table-columns}
 
-Above the table, click "Columns" to open a modal. This will show you a list of all the columns in the results table. You can add, delete and move basic key field columns or custom tags columns. With the same view, you can also stack events with any of the following functions:
+Above the table, click "Columns" to open the modal that shows you a list of all the columns in the results table. You can add, delete, and move basic key field columns or custom tag columns by which results are grouped.
 
 ### Stacking Functions
 
+You can also add any of the following functions as columns to stack events:
+
 - `avg(...)`
 - `count(...)`
```

Comment:
This doesn't seem to be what the UI reflects anymore. Now there's a function that looks like this: `count()` and it doesn't take a parameter. So I'm not sure if this just needs to be removed or if the paragraph below is no longer accurate.

Reply:
@imatwawana `count()` would be one of the few functions that do not take in a parameter. In circa 2019, it did take in a parameter; but this was changed. 

`count()` would need to be here; some paragraphs might need some re-wording.